tasks/src/main/java/org/example/aProblemAboutEquation/AProblemAboutEquation3.java

```diff
@@ -0,0 +1,79 @@
+package org.example.aProblemAboutEquation;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+
+/**
+ * https://codeforces.com/problemset/problem/174/A
+ */
+
+public class AProblemAboutEquation3 {
+    boolean numberOfFriendsVerifier(int numberOfFriends) {
+        return numberOfFriends >= 2 && numberOfFriends <= 100;
+    }
+
+    boolean millilitersOfTheDrinkInTheBottleVerifier(int millilitersOfTheDrinkInTheBottle) {
```

Comment:
nie moze to byc prywatne? Pytam bo nie doczytalem kodu

Reply:
Ta metoda akurat nie, bo jest wywoływana w testach. Jak zmieniam, to świeci się na czerwono...
Zmieniłem na private inną metodę:
private List<Float> aProblemAboutEquationWithFastScan()
oraz klasę statyczną:
private static class FastReader{}. 
Dopisałem main() i sprawdziłem - działają jako prywatne.